span.go

```diff
@@ -145,3 +160,11 @@ func CanonicalizeTraceAttributeKey(key string) (string, bool) {
 	}
 	return strings.ToLower(key), true
 }
+
+// StartChildSpan is a simple helper to start a child span given only its parent (per StartSpanOptions.Parent) and an operation name per Span.SetOperationName.
+func StartChildSpan(parent Span, operationName string) Span {
```

Comment:
A consolation prize since this PR gets rid of `Span.StartChild`... doesn't strictly need to be here, but it seems polite.